benchmark/fft_batch/fft_batch_config.cc

```diff
@@ -0,0 +1,55 @@
+#include "benchmark/fft_batch/fft_batch_config.h"
+
+#include "tachyon/base/console/iostream.h"
+#include "tachyon/base/containers/container_util.h"
+#include "tachyon/base/flag/flag_parser.h"
+
+namespace tachyon::benchmark {
+
+bool FFTBatchConfig::Parse(int argc, char** argv) {
+  base::FlagParser parser;
+
+  // clang-format off
+  parser.AddFlag<base::Flag<std::vector<size_t>>>(&exponents_)
+      .set_short_name("-k")
+      .set_required()
+      .set_help("Specify the exponent 'k's where the degree of poly to test is 2ᵏ.");
+  parser.AddFlag<base::BoolFlag>(&run_coset_lde_)
+      .set_long_name("--run_coset_lde")
+      .set_default_value(false)
+      .set_help("Run CosetLDE benchmark. Default is FFT benchmark.");
+  parser.AddFlag<base::Flag<bool>>(&check_results_)
+      .set_long_name("--check_results")
+      .set_help("Whether checks results generated by each fft_batch runner.");
```

Comment:
If you follow my idea, this isn't needed since it's implemented in `FlagParserBase`.